great_expectations/core/expectation_configuration.py

```diff
@@ -970,6 +972,8 @@ def __init__(
         self.success_on_last_run = success_on_last_run
         self._ge_cloud_id = ge_cloud_id
         self._expectation_context = expectation_context
+        if include_rendered_content:
+            self.rendered_content = {}
```

Comment:
Could we always set this attribute? I'd like to avoid conditional modifications of the shape of the object if possible.

Reply:
We can, but just note that this will mean the default behavior of `ExpectationValidationResult` will be to contain an empty `rendered_content` attribute. We could just modify the `__repr__` to hide it. Until I pull the feature flag out it might be best to leave it like this...

Reply:
It felt wrong to pass `include_rendered_content=False` and then have a new attribute called `rendered_content` anyway.

Reply:
It felt wrong to pass `include_rendered_content=False` and then have a new attribute called `rendered_content` anyway.

Reply:
Could it be a nullable attribute? Make it `None` if `False`?

Reply:
This attribute is no longer conditional, but will stay private `self._rendered_content` until the feature flag is pulled out.